src/main/resources/static/jhk.css

```diff
@@ -75,6 +75,11 @@ main {
     background-color: #c7c7c7
 }
 
+/* Help Keybind: goldenrod */
```

Comment:
The comment probably isn't necessary here.
Also, consider a more descriptive name than .button7.  Perhaps name it helpkeybind, if that's what it is for.  Then, no need for the comment.